### PR TITLE
Mattis

### PR DESCRIPTION
**Pull request by LinguList**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -20,7 +20,7 @@
         ]
     },
     install_requires=[
-        'pylexibank>=1.1.1',
+        'pylexibank>=2.0.0',
         'segments==2.0.2'
```
Comment (chrzyki) [inline setup.py]: Can we unpin this and have >=2.0.2 for segments?

Comment (tresoldi) [reply setup.py]: Do we even need a pinned version for `segments`? `pylexibank` already informs the system which version it needs (https://github.com/lexibank/pylexibank/blob/021f37481d37b472715033033773543273ed301f/setup.py#L39)

Comment (xrotwang) [reply setup.py]: Agree, no pinning please.

**Comment by chrzyki** [comment]
Merging this now to get a preliminary figure of the new CLICS stats.

**Pull request merged**